.circleci/config.yml

```diff
@@ -0,0 +1,144 @@
+---
+version: 2
```

Comment:
With CircleCI build added, makes sense to also include the build status badge to the README.